packages/expo-payments-stripe/README.md

```diff
@@ -21,6 +21,29 @@ For bare React Native projects, you must ensure that you have [installed and con
 expo install expo-payments-stripe
 ```
 
+- Include the config plugin in your `app.config.js` or `app.json`. Add the following props:
+  - `scheme` is the redirect URI that's used for returning to the app after an external payment. This should not match one of your existing schemes as it'll redirect to a special Activity on Android.
+  - `merchantId` an Apple ID that's required for making payments. This value should match the `merchantId` property used with `setOptionsAsync({ ... })` in your React code.
```

Comment:
`appleMerchantId`?

Reply:
this matches the prop name used in the API, makes it a bit easier imo.